### PR TITLE
JENKINS-26445 clone the DOM node used for overflow checking to avoid …

### PR DESCRIPTION
**Pull request by mrooney**
…unnecessary reflows and dramatically improve performance

This also simplifies the code a bit in that we don't need a try / finally block as we aren't modifying the original element.

### CODE REVIEW

File: war/src/main/webapp/scripts/hudson-behavior.js [hunk 1]
```diff
@@ -1953,6 +1953,8 @@ function getElementOverflowParams(element) {
     // wrapping is potentially happening, or not.
 
     // Force it to wrap.
+    var original = element;
+    element = element.clodeNode();
```
Comment (daniel-beck) [inline war/src/main/webapp/scripts/hudson-behavior.js]: Typo, or what's going on here?

**Comment by tfennelly** [comment]
Thanks for trying to fix this but I'm afraid this doesn't work so far due to a JS error (`clodeNode` is undefined). That explains why you're seeing a performance improvement (coz it's not doing anything).

screenshot 2015-09-16 08 55 20

Maybe try fixing that and see what happens.

Also note that there was some work done wrt pagination as a fix of this issue.  See #1641.

**Comment by tfennelly** [comment]
@mrooney what's happening with this PR?

**Comment by mrooney** [comment]
Hey @tfennelly, the original patch was a typo I made when porting it over, I did a quick sanity check after I had tested our local version more thoroughly, I just saw that the speed increased after applying the patch which just happened to be because of the typo. Anyway, updated it with correct spelling, I was less concerned given renewed interest in the pagination, but I suppose regardless, a speed up is a speed up!

**Comment by daniel-beck** [comment]
Since the sidepanel is now paginated, I think we can close this. @mrooney let us know if you disagree and this should be considered anyway.

**Comment by mrooney** [comment]
Definitely we can close it, thanks Daniel!

**Pull request closed without merging**